src/pages/LendingTeams/TeamSearchBar.vue

```diff
@@ -0,0 +1,47 @@
+<template>
+	<div class="tw-flex tw-gap-0.5 tw-flex-row">
+		<kv-text-input
+			id="team-search-bar"
+			placeholder="Search Lending Teams..."
+			v-model="queryString" class="tw-flex-1"
+		/>
+
+		<kv-button
+			variant="secondary"
+			@click="handleSearchTeams"
+			class="tw-flex-none"
+			v-kv-track-event="[
+				'teams',
+				'filter',
+				'teams-search',
+				queryString
+			]"
+		>
+			Search
+		</kv-button>
+	</div>
+</template>
+
+<script>
+import KvButton from '~/@kiva/kv-components/vue/KvButton';
+import KvTextInput from '~/@kiva/kv-components/vue/KvTextInput';
+
+export default {
+	name: 'TeamSearchBar',
+	components: {
+		KvButton,
+		KvTextInput,
+	},
+	inject: ['apollo'],
```

Comment:
I thought `cookieStore` was now required for apollo queries. It may not be though if these queries have no dependency on the kvbskt cookie. Also applies to teamLeaderboard and teamListing files...

Reply:
It's only really required for preFetched queries